### PR TITLE
Add Source Code and Docs to gource-log GitHub Action

### PR DESCRIPTION
**Pull request by IsaacMilarky**
## Problem

The gource-log repository was previously not ready to be out-bounded as a public DSACMS repo. 

## Solution

I have created a tier 3 repository using repo-scaffolder and then added the source code and required documentation to the repository. 

## Test Plan
`sudo act -a build`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,23 +15,30 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
...
-<!-- TODO: This section helps contributors understand any team structure in the project (formal or informal.) Encouraged to point towards the MAINTAINERS.md file for further details.-->
+1. Clone the repo
 
-### Building dependencies
+    `git clone https://github.com/IsaacMilarky/gource-log.git`
 
-<!--- TODO: This step is often skipped, so don't forget to include the steps needed to install on your platform. If you project can be multi-platform, this is an excellent place for first time contributors to send patches!-->
+2. Install the required packages and build dependencies:
 
-### Building the Project
```
Comment (natalialuzuriaga) [inline CONTRIBUTING.md:28]: "Building dependencies" and "Building the project" sections are required for Tier 3. Are these sections not applicable to the project?

**Review by natalialuzuriaga — CHANGES_REQUESTED** [review]
In addition, it looks like the `.github/codejson` directory is missing the `hooks` dir and the actual `code.json` file. Did you use repo-scaffolder to add docs to the project? If so, it looks like it didn't copy over all the needed files then which is something I'll have to fix.
https://github.com/DSACMS/repo-scaffolder/tree/main/tier3/%7B%7Bcookiecutter.project_slug%7D%7D/.github/codejson

One outstanding question but otherwise this is a solid start for this repo!!

File: README.md [hunk 2]
```diff
@@ -43,39 +56,84 @@ The following guide is for members of the project team who have access to the re
...
+This project adheres to PEP8 rules and guidelines whenever possible when accepting
+new contributions of Python code. Although, there are good reasons to ignore particular guidelines
+in particular situations. Further information on PEP8 can be found [here.](https://peps.python.org/pep-0008/)
+
+This project utilizes pylint as the primary linter for backend code, while eslint and prettier handle code formatting and linting for the frontend. Checks are implemented upon new pull requests into protected branches to ensure code quality and consistency.
+
+Python code quality checks are extremely useful for lowering the
+cost of maintenence of Python projects. Further information on Pylint can be found [here.](https://pylint.readthedocs.io/en/latest/)
+
+## Coding Style and Linters
```
Comment (natalialuzuriaga) [inline README.md]: "Code Style and Linters" is on here twice

**Review by natalialuzuriaga — CHANGES_REQUESTED** [review]
One last thing oops!

File: CONTRIBUTING.md (hunk 1 above)
Comment (IsaacMilarky) [reply CONTRIBUTING.md:28]: You don't really build the dependencies so much as install them. I should add a section for building the project though.

File: README.md (hunk 2 above)
Comment (IsaacMilarky) [reply README.md]: Whoops will fix

**Comment by IsaacMilarky** [comment]
> In addition, it looks like the `.github/codejson` directory is missing the `hooks` dir and the actual `code.json` file. Did you use repo-scaffolder to add docs to the project? If so, it looks like it didn't copy over all the needed files then which is something I'll have to fix. https://github.com/DSACMS/repo-scaffolder/tree/main/tier3/%7B%7Bcookiecutter.project_slug%7D%7D/.github/codejson
> 
> One outstanding question but otherwise this is a solid start for this repo!!

I did use repo-scaffolder to build a tier 3 repo. The missing hooks is def worth looking into

**Comment by natalialuzuriaga** [comment]
> > In addition, it looks like the `.github/codejson` directory is missing the `hooks` dir and the actual `code.json` file. Did you use repo-scaffolder to add docs to the project? If so, it looks like it didn't copy over all the needed files then which is something I'll have to fix. https://github.com/DSACMS/repo-scaffolder/tree/main/tier3/%7B%7Bcookiecutter.project_slug%7D%7D/.github/codejson
> > One outstanding question but otherwise this is a solid start for this repo!!
> 
> I did use repo-scaffolder to build a tier 3 repo. The missing hooks is def worth looking into

Oops ignore me I misread and turns out the repo has the hooks dir and the rest of the files!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -15,23 +15,30 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
...
+    - Docker
+    - act
+    - gource
+    
+## Building the Project
```
Comment (natalialuzuriaga) [inline CONTRIBUTING.md]: nit: Make this a level 3 header: ###

File: README.md [hunk 4]
```diff
@@ -24,16 +25,28 @@ TODO: Good to include since this is an agency-led project -->
...
+├── entrypoint.py
+└── tests
+├── .github
+│   ├── dependabot.yml
+│   └── workflows
+│       ├── build.yml
+│       └── major-release-num.yml
+```
 
 **{list directories and descriptions}**
```
Comment (natalialuzuriaga) [inline README.md]: Remove

**Review by natalialuzuriaga — DISMISSED** [review]
After resolving the nitpicks, this all LGTM!!!!

**Pull request merged**